<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>521</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>190</green>
        <blue>201</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>190</green>
        <blue>201</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>190</green>
        <blue>201</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>190</green>
        <blue>201</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: #B2A1B8;
	font-family: &quot;Times New Roman&quot;;
	font-size: 18px;
}</string>
   </property>
   <widget class="QLineEdit" name="NumberLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Plus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_A">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_B">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_C">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_D">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_E">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_F">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Rev">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Обращение введенного числа</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Sqr">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Возведение введенного числа в квадрат</string>
    </property>
    <property name="text">
     <string>SQR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Dot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Minus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Mul">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Div">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MemoryClear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Очистка памяти</string>
    </property>
    <property name="text">
     <string>MC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MemoryRead">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Чтение числа из памяти</string>
    </property>
    <property name="text">
     <string>MR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MemoryAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Добавление числа к числу в памяти</string>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MemoryWrite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Запись числа в память</string>
    </property>
    <property name="text">
     <string>MS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Sign">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Смена знака числа</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Delete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Удаление последнего введенного символа</string>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Clear_All">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Сброс всех операций и аргументов</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Clear_Last">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Очистка поля ввода с сохранением операций</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Основание системы счисления:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_p">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>16</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Execute">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Memory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_buffer">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
   </widget>
   <addaction name="menu_buffer"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_copy">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
